plugins/hypervisors/vmware/src/com/cloud/hypervisor/vmware/resource/VmwareResource.java

```diff
@@ -2253,7 +2255,18 @@ protected StartAnswer execute(StartCommand cmd) {
 
             // Since VM was successfully powered-on, if there was an existing VM in a different cluster that was unregistered, delete all the files associated with it.
             if (existingVmName != null && existingVmFileLayout != null) {
```

Comment:
Can you extract this IF body to a method?
This `execute` method is huge.

Reply:
@rafaelweingartner Sure. Will update.

Reply:
Perhaps you forgot to push the commit?